### ISSUE TITLE
[import/no-duplicates] Aliases support?

### ISSUE THREAD

**Issue by FezVrasta** [import-js/eslint-plugin-import#2693]
Let's say I have some aliases set so that I can directly import from the configured baseUrl option of my jsconfig.json.

```js
// This file is `components/test.js`

import 'modules/foo/bar';
import '../modules/foo/bar;
```

Here the two imports actually refer to the same file, but `import/no-duplicates` can't detect it.

Do you think the rule could be expanded to support this use case?

**Comment by FezVrasta** [comment]
Ok I realized it's supported already but the vite resolver I was using didn't throw an error while consuming my config so I didn't realize it wasn't being used.
